be/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+### Asana Link
+https://app.asana.com/0/1203341829928748/1203334927922401/f
+### Commands
+```
+cd backend
```

Comment:
- this will probably not work. move this to the right directory
![image](https://user-images.githubusercontent.com/73863685/201276634-caa3f8b8-2ba3-4a8a-ad2f-b9314b10d0a7.png)